src/plugins/maps_legacy/public/map/map_messages.js

```diff
@@ -110,39 +110,17 @@ export const createZoomWarningMsg = (function () {
             <FormattedMessage
               id="maps_legacy.opensearchDashboardsMap.zoomWarning"
               defaultMessage="You've reached the maximum number of zoom
-              levels. To zoom all the way in, upgrade to the
-              {defaultDistribution} of OpenSearch and OpenSearch Dashboards. You'll get
-              access to additional zoom levels for free through the {ems}.
-              Or, you can configure your own map server. Please go to
-              { wms } or { configSettings} for more information."
-              // TODO: [RENAMEME] Need valid URLs
+              levels. Please go to { wms } for more information."
```

Comment:
Do we need to leave the instructions about configuring your own map server? Is it still possible to access additional zoom levels through other means?

Reply:
This is a good point then can use their own.